core/types/tx_rip7560.go

```diff
@@ -293,3 +293,10 @@ type BundleReceipt struct {
 	GasPaidPriority     *big.Int
 	BlockTimestamp      uint64
 }
+
+type Rip7560TransactionDebugInfo struct {
```

Comment:
it is more "rejectedTransactionInfo".

Reply:
It is now, but I think we might want to expose some other data later as we work on a devnet.